packages/helm-charts/grafana/templates/ingress.yaml

```diff
@@ -3,7 +3,11 @@
 {{- $servicePort := .Values.service.port -}}
 {{- $ingressPath := .Values.ingress.path -}}
 {{- $extraPaths := .Values.ingress.extraPaths -}}
-apiVersion: networking.k8s.io/v1beta1
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
```

Comment:
Dunno exactly if there is a plan about how to handle ingressclasses and ingresses, but I think the object structure changes between `networking.k8s.io/v1` (follows this PR) and `extensions/v1beta1` (follows this for the backend:
```
  rules:
  - host: "22"
    http:
      paths:
      - backend:
          serviceName: "24"
          servicePort: "25"
        path: "23"
```
So I think this is really compatible with `extensions/v1beta1`.

Also, I don't know if a reference to `spec.ingressClassName` for selecting the ingressClass is needed ([annotations are deprecated](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation)) or do we rely on the default behavior always?

Reply:
Good catch. I've upgraded the ingress in [this PR](https://github.com/celo-org/celo-monorepo/pull/9475).